res/drawable/network_vbl_logo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="202.8775dp"
+    android:height="53.224998dp"
+    android:viewportWidth="202.8775"
+    android:viewportHeight="53.224998">
```

Comment:
Currently, all graphics use 64dp x 64dp. Could you maybe change for source SVG to 64px x 64px (can have transparent background parts for unused height) and recreate this XML? Then it should be 64dp automatically.